Tests fail on Windows

I just realized `mvn -B test` failes on Windows with:

    testParseSingleUserInfos(com.urswolfer.gerrit.client.rest.http.accounts.AccountsParserTest)  Time elapsed: 0.005 sec  <<< FAILURE!
    com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 3 path $
        at com.google.gson.JsonParser.parse(JsonParser.java:65)
        at com.urswolfer.gerrit.client.rest.http.common.AbstractParserTest.getJsonElement(AbstractParserTest.java:35)
        at com.urswolfer.gerrit.client.rest.http.accounts.AccountsParserTest.testParseSingleUserInfos(AccountsParserTest.java:70)
    Caused by: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 3 path $
        at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1574)
        at com.google.gson.stream.JsonReader.checkLenient(JsonReader.java:1423)
        at com.google.gson.stream.JsonReader.doPeek(JsonReader.java:546)
        at com.google.gson.stream.JsonReader.peek(JsonReader.java:429)
        at com.google.gson.JsonParser.parse(JsonParser.java:60)
        ... 36 more

So it seems this is because of the magic prefix (`)]}'`) in Gerrit's responses. Interestingly, I couldn't find any code that handles this magic prefix. So comes the test pass on Linux if the magic prefix is not stripped before the JSON string is passed to gson?

--- a/src/test/resources/.gitattributes
+++ b/src/test/resources/.gitattributes
@@ -1,0 +1,1 @@
+*.json	eol=lf

